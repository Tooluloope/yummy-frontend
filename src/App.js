import React from "react";
import "./App.css";
import {BrowserRouter as Router , Switch, Route} from "react-router-dom";
import { SignUp } from "./pages/auth/signup";
import { Login } from "./pages/auth/login";
import { Home } from "./pages/homepage";
import { UnAuthenticatedRoute } from "./components/routes/unauthenticatedRoutes";

function App() {

  return (
      <Router>
        <Switch>
          <UnAuthenticatedRoute path= '/login'>
            <Login />
          </UnAuthenticatedRoute>
          <UnAuthenticatedRoute path= '/signup'>
            <SignUp />
          </UnAuthenticatedRoute>
          <Route>
            <Home exact path='/'/>
          </Route>
         
          
        </Switch>
      </Router>
    
  );
}

export default App;
