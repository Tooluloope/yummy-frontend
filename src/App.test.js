import React from "react";
import { render } from "@testing-library/react";
import App from "./App";

import TestRenderer from "react-test-renderer";
import ShallowRenderer from "react-test-renderer/shallow";


// Basic Test with React-test-renderer
it("renders correctly react-test-renderer", () => {
  const renderer = new ShallowRenderer();
  renderer.render(<App />);
  const result = renderer.getRenderOutput();

  expect(result).toMatchSnapshot();
});
