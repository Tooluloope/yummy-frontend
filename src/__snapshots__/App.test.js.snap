// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly react-test-renderer 1`] = `
<BrowserRouter>
  <Switch>
    <UnAuthenticatedRoute
      path="/login"
    >
      <Login />
    </UnAuthenticatedRoute>
    <UnAuthenticatedRoute
      path="/signup"
    >
      <SignUp />
    </UnAuthenticatedRoute>
    <Route>
      <Home
        exact={true}
        path="/"
      />
    </Route>
  </Switch>
</BrowserRouter>
`;
